docs/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fixed a UI bug with fluxmonitor jobs where initiator params were bunched up.
+- Improved performance of OCR jobs to reduce database load
```

Comment:
Maybe add the new env var as well, and document the fact that the behavior of the parallelism env var has changed